Tasks/FileTransformV2/task.json

```diff
@@ -17,7 +17,7 @@
     ],
     "version": {
         "Major": 2,
-        "Minor": 179,
+        "Minor": 180,
```

Comment:
Could you please update version to 181?

Reply:
@egor-bryzgalov its still showing 179 in master branch and I have updated it to 180. can you please let me know the reason to update it to 181?

Reply:
@egor-bryzgalov done